src/collective/exportimport/import_content.py

```diff
@@ -422,14 +422,18 @@ def import_new_content(self, data):  # noqa: C901
             self.import_blob_paths(new, item)
             self.import_constrains(new, item)
 
-            self.global_obj_hook(new, item)
-            self.custom_obj_hook(new, item)
-
             uuid = self.set_uuid(item, new)
 
             if uuid != item.get("UID"):
+                # This happens only when we import external not Plone content that doesn't have a UID
+                logger.info("Created new UID for item {} with type {}".format(item["@id"],item["@type"]))
```

Comment:
In `logging` calls, we should keep using percentage formatting.  This avoids needless string interpolation when the logging is at a level that is ignored.

```suggestion
                logger.info("Created new UID for item %s with type %s.", item["@id"], item["@type"])
```